memory/src/main/scala/filodb.memory/format/ZeroCopyBinary.scala

```diff
@@ -115,7 +115,8 @@ object ZeroCopyBinary {
  */
 // scalastyle:off
 // FIXME Needs to use MemoryAccessor
-final class ZeroCopyUTF8String(val base: Any, val offset: Long, val numBytes: Int) extends ZeroCopyBinary {
+final class ZeroCopyUTF8String(val base: Any, val offset: Long, val numBytes: Int
+                  ) extends ZeroCopyBinary with Serializable {  // TODO(a_theimer): does this actually work?
```

Comment:
Is this secretly breaking something?

Reply:
Why ``Serializable``?

Reply:
Just so I can use byte/object input/output streams to serialize the `Map[ZeroCopyUTF8String, CardCounts]` I use to store cardinality data.

Reply:
Removed this change in most recent commit